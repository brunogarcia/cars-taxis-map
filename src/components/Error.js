import React from 'react';
import AlertIcon from '@material-ui/icons/Announcement';
import './Error.css';

const Error = () => {
  const message = 'Looks like there was a problem';

  return (
    <div className="Error-main">
      <p>{message} <AlertIcon /></p>
    </div>
  );
};

export default Error;
