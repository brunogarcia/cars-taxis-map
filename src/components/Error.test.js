import React from 'react';
import renderer from 'react-test-renderer';
import Error from './Error';

test('renders correctly', () => {
  const component = renderer.create(<Error />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
