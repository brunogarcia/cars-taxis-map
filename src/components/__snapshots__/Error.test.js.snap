// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Error-main"
>
  <p>
    Looks like there was a problem
     
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-1"
      color={undefined}
      focusable="false"
      viewBox="0 0 24 24"
    >
      <g>
        <path
          d="M20 2H4c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-7 9h-2V5h2v6zm0 4h-2v-2h2v2z"
        />
      </g>
    </svg>
  </p>
</div>
`;
